### PR TITLE
Edit page next steps

### PR DESCRIPTION
**Pull request by weliveindetail**
New features and fixes for the edit page

### CODE REVIEW

File: new-website/src/components/location/edit-form.jsx [hunk 1]
```diff
@@ -21,15 +29,112 @@ const useStyles = makeStyles((theme) => ({
...
+
+// Resolve coordinates for given address with LocationIQ
+async function requestLatLng(address) {
+  const sleep = async (ms) => {
+    await new Promise((wakeup, _) => setTimeout(wakeup, ms));
+  };
+  const tryResolve = async (address, retries) => {
+    const STATUS_CODE_RATE_LIMIT_EXCEEDED = 429
+    for (let i = 0; i < retries; i++) {
+      // TODO: Easy way to hide our key? (not urgent, it's a free account)
```
Comment (c0un7-z3r0) [inline new-website/src/components/location/edit-form.jsx:86]: Client side hiding of key is not possible ... if you really want to find out the final key then you just check the request we do in you browser. Either we "hide" it in a function (server side) or we just accept this risk

File: new-website/src/components/location/edit-form.jsx [hunk 2]
```diff
@@ -82,12 +243,77 @@ export const EditForm = ({ docId, doc, onError }) => {
...
+          <img src={MarkerIconImmunhelden} className={classes.markerIcon} alt="icon-immunhelden" />
+        )}
+        {(state.type === "new") && (
+          <img src={MarkerIconNew} className={classes.markerIcon} alt="icon-new" />
+        )}
+        <FormControl className={classes.smallField} style={{flex: 1}}>
+          <InputLabel id="demo-simple-select-helper-label">{formatMessage({ id: "partnerLocation_Type" })}</InputLabel>
+          <Select name="type" value={state.type} onChange={handleChange}
+                  labelId="demo-simple-select-helper-label">
+            <MenuItem value="plasma">{formatMessage({ id: "partnerLocation_TypePlasma" })}</MenuItem>
```
Comment (c0un7-z3r0) [inline new-website/src/components/location/edit-form.jsx:309]: do you really want to hardcode alle options and not just get them from firestore?

Comment (weliveindetail) [reply new-website/src/components/location/edit-form.jsx:309]: I think it's fine for now.

File: new-website/src/components/location/edit-form.jsx (hunk 1 above)
Comment (weliveindetail) [reply new-website/src/components/location/edit-form.jsx:86]: Yes, makes sense. I will clean up this and other TODOs in a follow-up PR.

**Comment by weliveindetail** [comment]
Rebased after landing https://github.com/ImmunHelden/ImmunHelden.de/pull/178

**Pull request merged**